docker/types/services.py

```diff
@@ -280,6 +282,8 @@ def __init__(self, target, source, type='volume', read_only=False,
                 volume_opts['Labels'] = labels
             if driver_config:
                 volume_opts['DriverConfig'] = driver_config
+            if subpath:
+                volume_opts['SubPath'] = subpath
```

Comment:
I think this should be written as `Subpath` here, right @vvoland?

Reply:
Yeah, it is `Subpath` instead of `SubPath`: [moby#45687](https://github.com/moby/moby/pull/45687)